### DISCUSSION TITLE
part-ii/2017-65/

### DISCUSSION THREAD

**Discussion by tripos-comments** [tripos-education/maths-tripos-questions#148] [Question comments]
Part II 2017 Further Complex Methods: Paper 2, Section II, E 

https://questions.tripos.org/part-ii/2017-65/

**Comment by acornnroca** [comment]
<details>
<summary>A and B</summary>


$$A = \frac{ \Gamma(c) \Gamma(c-b-a)}{\Gamma(c-a) \ Gamma(c-b)$$

$$B = \frac{ \Gamma(c) \Gamma(a+b-c)}{\Gamma{a} \Gamma{b}$$


</details>
